### ISSUE TITLE
As an FS Administrator I need to be able to edit the dates and times for Christmas cutting areas for the ARNF so that people can maintain flexibility in my program and the public can have updated information.

### ISSUE THREAD

**Issue by ASprinkle** [nciinc/fs-permit-platform#410]
During sprint planning it was decided to have a separate story that specifically addresses FS Admin ability to update Christmas tree cutting area dates and times.  This is a different story from the one that addresses editing Christmas tree sales season dates and times.

**Comment by npage123** [comment]
@bryanmcfadden @JaneZC   Here's a mockup of what the admin could see when selecting "Christmas Trees admin" from the header. Thoughts? Not sure which page would be the default home though.
![410-01](https://user-images.githubusercontent.com/12956624/36285593-cc4e08ec-1271-11e8-806c-0dadf53f5456.jpg)

**Comment by sadlerw** [comment]
mockup of the form when the forest has no cutting areas or districts with individual times - I need wording for the sentence "BLAH does not have any districts or cutting areas
          with start and end times different from the forest season start and end dates." please @JaneZC 

<img width="688" alt="screen shot 2018-02-16 at 9 06 11 am" src="https://user-images.githubusercontent.com/2058839/36313909-c64732de-12f8-11e8-9668-a7776e679908.png">

**Comment by sadlerw** [comment]
Same problem here with the legend - it needs something shorter:
<img width="550" alt="screen shot 2018-02-16 at 9 07 36 am" src="https://user-images.githubusercontent.com/2058839/36313965-f0b326fe-12f8-11e8-82ae-93f4f6dd3f35.png">

**Comment by sadlerw** [comment]
here's the form - do these look ok @JaneZC @bryanmcfadden @npage123 ?

<img width="561" alt="screen shot 2018-02-16 at 9 09 49 am" src="https://user-images.githubusercontent.com/2058839/36314058-405d5d1e-12f9-11e8-987e-395a0015997c.png">

**Comment by sadlerw** [comment]
maybe this?

<img width="600" alt="screen shot 2018-02-16 at 9 11 45 am" src="https://user-images.githubusercontent.com/2058839/36314129-759a1cba-12f9-11e8-8b18-844b90526a86.png">

**Comment by npage123** [comment]
I'm thinking we should re-order the dates and times to start date, end date, start time, and end time. Thoughts @sadlerw @JaneZC @bryanmcfadden ?

**Comment by sadlerw** [comment]
That's a shared component so it would change for the intake side too.  You should talk to whoever set it up for them and figure out why they got it in the wrong order :-P

**Comment by ASprinkle** [comment]
@sadlerw @JaneZC ideas regarding the text descriptions: "Select a cutting area to update the dates and times for that cutting area."
and for the Forest that doesn't have cutting areas: "XXNF does not have special cutting areas with different dates or times. The season dates are forest wide"

**Comment by sadlerw** [comment]
going to create a new story for the admin menu stuff

**Comment by MelissaBraxton** [comment]
Looking good! One minor thing (in addition to the content change Amber recommends in the above comment): Fix the district/cutting area names so that they are not all caps and all one word.

**Comment by tram** [comment]
Recommend moving "To update, remove, or create a new cutting area…" to below the cutting area choices (like it is for "If your forest is not listed…" in the choice above). Also recommend adding Oxford comma after "remove" in "To update, remove, or create a new cutting area…" to make a little clearer. Other than that, looks good to me!

**Comment by MelissaBraxton** [comment]
Heads up @sadlerw - As of 3/2, this does not appear to be working on staging. Also found a minor bug in the radio button state. Selected, it shows up like this: 

<img width="452" alt="screen shot 2018-03-02 at 4 23 20 pm" src="https://user-images.githubusercontent.com/20651166/36927894-acd6423e-1e35-11e8-949b-4499f4dea82e.png">

**Comment by sadlerw** [comment]
good catch - broken by a typo in a refactor - I added a e2e spec to catch it in future.  The specs all passed because the page was still working - you just couldn't see it! the e2e will catch that now.

**Comment by sadlerw** [comment]
@MelissaBraxton if you look at the radio button section here: https://nciinc.github.io/fs-fork-uswds-site/components/form-controls/ you'll see it's the same.  Does it need to change?
